src/and-cli-ls.ts

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+
+import { CommandRunner } from "./modules/command-runner";
+import { CommandDefinitions } from "./modules/command-definitions";
+import program from "commander";
+import { Options } from "./constants/options";
+import { ListCommands } from "./modules/list-commands";
+
+// -----------------------------------------------------------------------------------------
+// #region Constants
+// -----------------------------------------------------------------------------------------
+
+const { DEFAULT_OPTIONS } = ListCommands;
+const {
+    includeHelp: DEFAULT_INCLUDE_HELP,
+    indent: DEFAULT_INDENT,
+    prefix: DEFAULT_PREFIX,
+    skipCache: DEFAULT_SKIP_CACHE,
+    useColor: DEFAULT_USE_COLOR,
+} = DEFAULT_OPTIONS;
+
+// #endregion Constants
+
+CommandRunner.run(async () => {
```

Comment:
For sure, I like the brevity of it and I don't foresee a need to wrap `ls` at this point, it could still cause some confusion. I originally started with just `commands` and that didn't feel quite right, maybe `list-commands`?

Reply:
Or a middle ground would be `list` like mentioned below.